src/App.jsx

```diff
@@ -54,25 +56,39 @@ function App() {
       link: bookmark.link,
       description: bookmark.description,
     }
+    toast.success("Bookmarked", {
```

Comment:
move this function to some other place like utils.
make it easier to use it like 
```
toastSuccess("Bookmarked" )
```

the first msg parameter will be required and the second object will be optional,
its default might be 
```
{ autoClose: 2000,
      closeOnClick: true,
      pauseOnHover: true,
      draggable: true,
      theme: "colored",
      }
```